tests/lib/helpers.js

```diff
@@ -218,6 +232,13 @@ function assertUATrackingPageVar(test, key, val) {
 }
 
 
+function assertNotVisible(selector) {
```

Comment:
Okay, I did some digging. Both slimer and phantom look the same in screenshots.

We're using casperjs 1.1.0-beta3 so the code being run is https://github.com/n1k0/casperjs/blob/1.1-beta3/modules/clientutils.js#L156-L171.

This is slightly different in that it doesn't check for `inline-block`. And running this in phantom and slimer yields different results:

```js
casper.echo(JSON.stringify(casper.evaluate(function() {
    var elem = window.document.querySelector(
        '.account-settings input[name="display_name"]');
    var style = window.getComputedStyle(elem, null);
    return [style.visibility, style.display, elem.clientHeight, elem.clientWidth];
})));

// PhantomJS: ["visible","inline-block",0,0]
// SlimerJS: ["visible","inline",0,0]
```

So what I take from this is that visibility checks only work on the specific element you're checking. In the future, all `display: inline-block;` elements will be treated as visible. Which seems like a bug.

I don't see anything setting `display` on the inputs so I guess WebKit treats it as `inline-block` by default and Gecko as `inline`.